tests/testoverrideconfig/apps.py

```diff
@@ -0,0 +1,9 @@
+from django_dramatiq.apps import DjangoDramatiqConfig, RATE_LIMITER_BACKEND
+
+
+class GroupsCallbacksDjangoDramatiqConfig(DjangoDramatiqConfig):
+
+    @classmethod
+    def middleware_groupcallbacks_kwargs(cls):
+        import pdb; pdb.set_trace()
```

Comment:
I can't reach this code because I'm unable to override the django_dramatiq config in the tests.